array_api_tests/pytest_helpers.py

```diff
@@ -421,6 +421,15 @@ def assert_fill(
         assert xp.all(xp.equal(out, xp.asarray(fill_value, dtype=dtype))), msg
 
 
+def _has_signbit() -> bool:
```

Comment:
A small comment/docstring explaining that this function exists because `signbit` can be available but not implemented would help I think. Possibly worth re-naming the function too/instead, something like `_has_functional_signbit()`.